runtime/CMakeLists.txt

```diff
@@ -65,11 +65,19 @@ if(BUILD_QIR_STDLIB_FROM_SRC OR APPLE)
     FetchContent_MakeAvailable(qir_runner)
 
     execute_process(
-        COMMAND cargo build --release --package qir-stdlib --target-dir ${qir_runner_BINARY_DIR}
+        COMMAND cargo build --release --package qir-stdlib --target-dir ${qir_runner_BINARY_DIR} RESULT_VARIABLE cargo_found_status
         COMMAND ${CMAKE_COMMAND} -E copy ./stdlib/include/qir_stdlib.h ${qir_runner_BINARY_DIR}/release/qir_stdlib.h
         WORKING_DIRECTORY ${qir_runner_SOURCE_DIR}
     )
 
+    if(NOT cargo_found_status EQUAL "0")
+        message(STATUS "-- To build runtime from source, you also need \"cargo\"")
+        message(STATUS "-- and the \"llvm-tools-preview\" rustup component.")
+        message(STATUS "-- Run the following command to install these dependencies:")
+        message(STATUS "-- $ curl https://sh.rustup.rs -sSf | sh && source \"$HOME/.cargo/env\" && rustup component add llvm-tools-preview")
+        message(FATAL_ERROR "Not found cargo")
```

Comment:
```suggestion
        message(FATAL_ERROR "Missing cargo installation")
```

Reply:
Actually I don't know about having these messages be split, it might be easy to overlook the STATUS messages above?

Reply:
@dime10 lmk what you think about [f9d24b4](https://github.com/PennyLaneAI/catalyst/pull/393/commits/f9d24b4cf6bb9eee472885adeac2962cfa6c2b68)?

Reply:
Looks good 👍